src/SDL_gpu_shadercross.c

```diff
@@ -1547,22 +1558,8 @@ SDL_GPUComputePipeline *SDL_ShaderCross_CompileComputePipelineFromSPIRV(
 
 bool SDL_ShaderCross_Init(void)
 {
+    #ifdef _GAMING_XBOX
```

Comment:
I think you can use `#if defined(SDL_PLATFORM_XBOXONE) ||  defined(SDL_PLATFORM_XBOXSERIES)`